include/fmi4cpp/fmu_instance_base.hpp

```diff
@@ -60,6 +60,11 @@ class fmu_instance_base : public virtual fmu_instance<model_description>
         , modelDescription_(modelDescription)
     {}
 
+    std::shared_ptr<const fmi_library> get_library() const
```

Comment:
Is it necessary to expose this type? It is not used by this PR either.

Reply:
Looks like a leftover from earlier experimentation. I'll remove it.

Reply:
Done